request_test.go

```diff
@@ -114,7 +114,9 @@ func TestGetRequestInput(t *testing.T) {
 		}
 
 		if assert.NoError(t, err, "getRequestInput(%v, %v) should not fail", tt.inline, tt.file) {
-			assert.Equal(t, tt.want, got, "getRequestInput(%v, %v) mismatch", tt.inline, tt.file)
+			contents, err := ioutil.ReadAll(got)
+			assert.NoError(t, err)
```

Comment:
nit: `require.NoError(t, err)`

If this check fails, the following `assert.Equal` will likely fail as well. To avoid having multiple consecutive failures, use `require` here to bail out early.